dbmongo/js/globals.ts

```diff
@@ -26,6 +26,7 @@ type NAF = {
 
 type Scope = "etablissement" | "entreprise"
 
+// Données importées pour une entreprise ou établissement, sur une période donnée.
```

Comment:
```suggestion
// Données importées pour une entreprise ou établissement
```
En l'occurence, un objet `Company data values` aura *toutes* les données importées concernant cette entreprise ou établissement (historisés dans `batch`)